Cake.Recipe/Content/build.cake

```diff
@@ -234,39 +251,16 @@ public void CopyBuildOutput()
             continue;
         }
 
-        if(parsedProject.OutputPath == null || parsedProject.RootNameSpace == null || parsedProject.OutputType == null)
+        if(parsedProject.OutputPath == null || parsedProject.AssemblyName == null || parsedProject.OutputType == null)
```

Comment:
RootNamespace is populated for all parsed project types
for netcore where it isn't a mandatory element it has the following fallback order
RootNamespace > AssemblyName > ProjectName